forte/ontology_specs/clinical.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "clinical",
+  "imports": [
+    "forte/ontology_specs/base_ontology.json"
+  ],
+  "definitions": [
+    {
+      "entry_name": "ft.onto.clinical.ClinicalEntityMention",
```

Comment:
It is better to extend `ft.onto.base_ontology.EntityMention`, and then you don't have to provide the attribute `cliner_type`, simply use the `ner_type` attribute from the root project.

Reply:
Could you extend `ft.onto.base_ontology.EntityMention` instead?

Reply:
Fixed. Pls check again.